HackYourFuture-CPH/team26-checkin
# Frontend landing page

# Implementation of the landing page 
How to test?
Run the client and see the result in localhost:3000/teams-check-in

Checklist
[Yes] I have performed a self-review of my own code
[Yes ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
[Yes ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
[Yes ] This PR is ready to be merged and not breaking any other functionality

![Uploading landingpage.png…]()
